### DISCUSSION TITLE
API r=user/-/state/com.google/starred is not working in my script

### DISCUSSION THREAD

**Discussion by cc13com** [FreshRSS/FreshRSS#5129] [Q&A] [answered]
**Describe the bug**
With my python script I try to remove a "star" from an item but it does not work.

**To Reproduce**

I wrote my own python script to work with FreshRSS. The script login to my instance, get all starred items and should remove the star from this items. For testing I created a curl command:

```
curl -X "POST" "https://MY_DOMAIN/api/greader.php/reader/api/0/edit-tag?i=1234a4321&r=user%2F-%2Fstate%2Fcom.google%2Fstarred" \
     -H 'Authorization: GoogleLogin auth=user_name/hash_value' 
```

In the command above I replaced the domain, item-tag and auth-token just for the posting here.

If I run the command I get an "OK" back, but the item is still starred.

I know there is a lot of documentation out there and for me the command looks like described in the docs.

**Comment by Alkarex** [comment, marked as answer]
Hello,
On the road at the moment. Those parameters (`i`, `r`) must be passed by POST, not by GET as you do now.
In other words, the parameters must be a part of the POST payload, not a part of the query string.

Something like that (not tested):

```sh
curl 'https://freshrss.example.net/api/greader.php/reader/api/0/edit-tag' \
  -H 'Authorization: GoogleLogin auth=user_name/hash_value' \
  -d 'i=1234a4321' -d 'r=user/-/state/com.google/starred'
```

**Comment by cc13com** [comment]
@Alkarex thanks, your example works. Now I have to adapt that in my client "RapidAPI" where I got my posted example.
